src/analyze/analyze.c

```diff
@@ -1845,13 +2014,17 @@ static int help(int argc, char *argv[], void *userdata) {
                "  -M --machine=CONTAINER   Operate on local container\n"
                "     --order               Show only order in the graph\n"
                "     --require             Show only requirement in the graph\n"
+               "     --failure             Show only on-failure dependencies in the graph\n"
+               "     --triggers            Show only triggers in the graph\n"
```

Comment:
so, before we keep adding switches over switches like this I think we should rework this so that there's instead:

```--dependency-kind=failure|triggers|order|require``` and then keep the old `--order` and `--require` only in place for compatibility, but hide it in the --help text and in the man page.

Reply:
done